db/sqlglue.c

```diff
@@ -831,6 +831,7 @@ static int ondisk_to_sqlite_tz(struct dbtable *db, struct schema *s, void *inp,
         genid = rrn;
 
     if (genid) {
+        memset(&m[fnum], 0, sizeof(Mem));
```

Comment:
This shouldn't really matter, right? Since the type is `MEM_Int`, we only need `Mem.u.i` in `SerialPut`.

Reply:
Agreed. Nevertheless, safer to initialize now than expose ourselves to uninitialized memory bugs in the future , when/if things change

Reply:
I think the intention here is to only access what we mean (u.i when MEM_Int is set) and any other usage is a bug -- in fact memsetting could cover up for such bug. It can also be less performant, altho I can't tell with what frequency this is called.